source/libs/utils.ts

```diff
@@ -204,3 +205,22 @@ export function getLatestVersionTag(tags: string[]): string {
 
 	return latestVersion;
 }
+
+export function parseRoute(pathname: string) {
```

Comment:
This is weird. Normally TS complains if this type is missing
```suggestion
export function parseRoute(pathname: string): string {
```

Reply:
Really this should not be part of this PR. I Cherry picked it until #3083 is merged. I will fix it there

Reply:
Let's merge this one first. It's done